### DISCUSSION TITLE
Settings not available in the Top menubar only mode

### DISCUSSION THREAD

**Discussion by BertLvB** [rsayfutdinov/Pikka#59] [General]
When the app in menubar only is selected, it doesn't seem possible to access the apps settings anymore. This also means that reverting this setting doesn't seem possible? In addition, if the color library is empty, selecting the color library from the menu bar icon does not bring up the main window.
Also is it not possible to delete a singe color from a palette in the color library
<img width="321" alt="Scherm­afbeelding 2024-06-07 om 09 18 38" src="https://github.com/rsayfutdinov/Pikka/assets/171877077/5ebc5a24-3305-4a4a-9c08-1a68ea0429ce">

**Comment by mmabrouk** [comment]
I have the same problem. I cannot access the setting anymore.

**Comment by rsayfutdinov** [comment]
Thanks for letting me know. An update is coming soon.

1. You can now use `CMD+,`  inside of `Color Library` to open the app settings.
2. To delete a single color from the library use "swipe to delete".

**Reply by mmabrouk** [reply]
Thanks @rsayfutdinov for the quick answer!
Using `CMD + ,` solves the problem.

**Comment by BertLvB** [comment]
The solutions you gave doesn’t work at all. Please note that not everyone has a touch screen, most of are using a mouse.
CMD+ results in a lot of “pings” nothing further

Dissapointed in the new version, the older one was better.

Maybe the update will fix it all, but I’m sceptic.

Regards,
Bert.


Op 18 jun 2024, om 19:50 heeft ruslan ***@***.***> het volgende geschreven:

**Comment by rsayfutdinov** [comment]
`CMD+,` not just CMD+ 
You can swipe using a trackpad or a mouse.

**Comment by BertLvB** [comment]
Indeed CMD+,   works te get the settings back.
But swing with a mouse, could you tell me how to?
I’m a Mac user for 20 years, website building company owner, so I know how to work a Mac.
But you explanation in a few words is very disturbing.

**Reply by rsayfutdinov** [reply]
If you have a Macbook swipe on the trackpad with two fingers to delete a color.
If you are using an Apple Magic Mouse swipe with one finger on the magic mouse, like the way you scroll horizontally.

You can also go to the macOS System Settings, then click Mouse or Trackpad in the sidebar and discover the gestures.

**Reply by BertLvB** [reply]
Well, i'm not using a trackpad or magic mouse but a vertical mouse with no swipe options. I never scroll horizontaly, call me old fashioned but thats the way i work.

**Comment by stephansann** [comment]
+1 - cannot access the settings after setting the app to "Menubar only".
Had to uninstall the app and install it anew to get to the preferences :-(
